CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ include(CheckIncludeFile)
 include(CheckIncludeFiles)
 include(CheckCXXSourceCompiles)
 include(CheckSymbolExists)
+include(GNUInstallDirs)
```

Comment:
Just saw "GNU", so I'm wondering if this has any flow-on effects on @FeralChild64 WIP CMake install work?

https://github.com/dosbox-staging/dosbox-staging/pull/4435

Reply:
Yes - I'm including this module too :)